src/coreclr/tools/Common/Compiler/HardwareIntrinsicHelpers.cs

```diff
@@ -269,11 +229,16 @@ private static class Arm64IntrinsicConstants
             public const int Rcpc = 0x0100;
             public const int Rcpc2 = 0x0200;
             public const int Sve = 0x0400;
+            public const int Sve2 = 0x0800;
 
             public static void AddToBuilder(InstructionSetSupportBuilder builder, int flags)
             {
                 if ((flags & AdvSimd) != 0)
+                {
+                    // We need to explicitly track AdvSimd as some Linux machines can
+                    // still be encountered without it and we need to fail to launch
```

Comment:
I do not think that this is going to result into "nice" launch failure.

Reply:
This is what currently causes `nativeaot\runtime\startup.cpp` to print:
> "The required instruction sets are not supported by the current CPU."

However, based on your above comment on the JIT having been compiled assuming `AdvSimd` support itself, I agree it should be unnecessary and users may end up with a fault due to invalid instruction usage instead. I'll remove this since I need to add back in the R2R flags anyways